pages/tutorial.js

```diff
@@ -77,11 +80,12 @@ const tutorialView = (state, prev, send) => {
     },
 
     panels: [
-      blocklyHtml,
       [
-        gameHtml,
-        goalProgressHtml
-      ]
+        instructionHtml,
+        goalHtml
+      ],
+      blocklyHtml,
+      gameHtml
     ]
   })
```

Comment:
That looks like a nice idea! 
You should add that to the component, I seriously hardly understand something in the panel-group file. (also there is very few documentation besides the default behaviour of that module)

I adjusted the instruction panel to look like you suggested.